### ISSUE TITLE
Warning when using shortener with Rails 6

### ISSUE THREAD

**Issue by pepibumur** [jpmcgrath/shortener#124]
Using Shortener with the latest Rails version, 6.0, I get the following warning by Rails:

```
DEPRECATION WARNING: Initialization autoloaded the constant Shortener::ShortenerHelper.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload Shortener::ShortenerHelper, for example,
the expected changes won't be reflected in that stale Module object.

This autoloaded constant has been unloaded.

Please, check the "Autoloading and Reloading Constants" guide for solutions.
```

**Comment by hwhelchel** [comment]
Previous discussion here: https://github.com/jpmcgrath/shortener/issues/117

This has been fixed in the develop branch: https://github.com/jpmcgrath/shortener/commit/bb9001447d21e74b8248deb1010cf37c8280d417#diff-2d34f1c6ac3aef84f8d19ed465268c9dR137

It needs to be merged into master and a new release cut. @craigmcnamara can we get a new release cut please?

**Comment by ssaunier** [comment]
Maybe it's another error? I just added this to my `Gemfile`:

```bash
gem 'shortener', github: 'jpmcgrath/shortener', branch: 'develop'
```

When running `rails s` I get:

```bash
DEPRECATION WARNING: Initialization autoloaded the constants Shortener::ShortenerHelper and RedirectUriValidator.
```

**Comment by hwhelchel** [comment]
@ssaunier hmm you might be right. It's probably related to https://github.com/fxn/zeitwerk

The autoloading semantics have changed in rails 6 some.

**Comment by hwhelchel** [comment]
@jpmcgrath ping

**Comment by hwhelchel** [comment]
@craigmcnamara any thoughts on this?

**Comment by ajsharp** [comment]
@hwhelchel @jpmcgrath Is anyone interested in maintaining an active fork of this project?

**Comment by hwhelchel** [comment]
@fschwahn do you want to maintain an active fork of this project?

**Comment by fschwahn** [comment]
I'd rather not fork this. Maybe @jpmcgrath wants support with this repo, I'd be willing to help.

**Comment by hwhelchel** [comment]
Cool he hasn't responded on github. Here's his listed email address dev@jamespmcgrath.com and it looks like he is on LinkedIn if you want to try reaching out to him through either of those channels to get admin access to the repo.

**Comment by somezack** [comment]
Just curious if this got any further. @fschwahn, did you reach out? If not, I can.

**Comment by fschwahn** [comment]
No, please do @somezack

**Comment by jpmcgrath** [comment]
I released v0.8.1 tonight to address this issue (releasing #127). Apologies for the silence and hassle you may have experienced in having to run off a unofficial branch.

Very happy to have help in maintaining this project. If people are interested in being added as a collaborator to the repo, please let me know in this thread and I can add those people that have an established a trustworthy looking presence on the web. Adding people as an admin to the project is definitely something I would consider after we have established a level of trust.

**Comment by fschwahn** [comment]
As I said, I'd be willing to help. Would be awesome if CI would run again, that would make helping easier for sure!

**Comment by jpmcgrath** [comment]
Ok, travis is now running again on the repo. It appears to be erroring on the Ruby 2.2 and Rails 6 build, along with some other combos. It will need to be configured so that Rails versions only run builds with appropriate ruby versions.

**Comment by fschwahn** [comment]
@jpmcgrath  I added #133 to get CI green

**Comment by jpmcgrath** [comment]
Excellent @fschwahn. I have merged #133 and added you as a collaborator on the repo.
